modules/spades/spades.nf

```diff
@@ -14,7 +14,7 @@ process SPADES {
     tuple val(genomeName), path(genomeReads)
 
     output:
-    path("*_contigs.fasta")
+    tuple val(genomeName), path("*_contigs.fasta")
```

Comment:
@Mxrcon this is a good idea, however with this you need to change the places where `SPADES` output is being used in the `main.nf` and as a test, try to run the stub version and share the workflow :)

Reply:
I solved this using the emit: function to create 2 named outputs, one named `QUAST` and other `PROKKA`, I've also executed 2 workflows that uses spades : `BASE` and `SPADES_WF` could you check these tower executions that i shared with you on tower: `insane_northcutt` and `astonishing_tesla`?